packages/init/init.ts

```diff
@@ -33,37 +33,38 @@ function mapOptionsToTransform(config) {
  * and writes the file
  */
 
-module.exports = function runTransform(webpackProperties, action) {
+export default function runTransform(webpackProperties, action: string) {
```

Comment:
This could be also done like `(webpackProperties: any, action: string): any`. I prefer not to bloat code with `any`.

If we don't want to use `any`, then we can explicitly define interface/type of whole `webpackProperties` object.

Reply:
use any here, it's fine.

Reply:
We should avoid the use of `any`. Maybe we can leave a TODO to come back later once we have the interface. `any` is evil, it makes the use of a type checker basically pointless.

Reply:
ok